Angular/src/app/paradigm-list/paradigm-list.component.html

```diff
@@ -1,6 +1,7 @@
 <div class="flex-container">
-	<div class="top-item"><button (click)="goBack()" class="button"><i class="material-icons">arrow_back</i></button></div>
-	<div class="top-item"><button (click)="deleteLanguage()" class="button" id="delete"><i class="material-icons">delete</i></button></div>
+	<div class="top-item"><button (click)="goBack()" class="button" title="Go Back"><i class="material-icons">arrow_back</i></button></div>
```

Comment:
The title addition is great, the pop-up text when it is hovered over is extremely helpful!